yarn.lock

```diff
@@ -701,7 +701,18 @@
   dependencies:
     regenerator-runtime "^0.13.2"
 
+<<<<<<< HEAD
```

Comment:
Merge conflict here.

If it helps feel free to delete the file and `yarn install` from scratch